build.psm1

```diff
@@ -954,7 +954,7 @@ function Install-Dotnet {
             # This is the library shipped with .NET Core
             # This is allowed to fail as the user may have installed other versions of dotnet
             Write-Warning ".NET Core links the incorrect OpenSSL, correcting .NET CLI libraries..."
-            find $env:HOME/.dotnet -name System.Security.Cryptography.Native.dylib | % { sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib $_ }
+            find $env:HOME/.dotnet -name System.Security.Cryptography.Native.OpenSsl.dylib | % { sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib $_ }
```

Comment:
Yes, this is exactly what this change is about. The version of dotnet that is currently pinned uses two libs for crypto: Apple and OpenSSL. It was not the case for previous versions. The crypto package that we **consume** was split in two for a while already. These two patching logics are independent from the process point of view, but this changes makes the lines called exactly the same.